pcsx2/IopMem.cpp

```diff
@@ -490,3 +490,14 @@ void __fastcall iopMemWrite32(u32 mem, u32 value)
 		}
 	}
 }
+
+std::string iopMemReadString(u32 mem, int maxlen)
```

Comment:
Well maybe the default's too high, given that these will be going on the stack often.